Verify_Registration_BQ-77.py

```diff
@@ -15,25 +14,24 @@
 my_account=driver.find_element(By.LINK_TEXT, "My account")
 my_account.click()
 
-    
+#Create random email ids
+letters = string.ascii_letters
+length=10
+result = ''.join(random.choice(letters) for _ in range(length))
+email=result+"@gmail.com"
+print(f"The email created is: {email}")
+
 #Enter the test data to register (later write a program to randomly create email ids so you dont get the "User already exists" error)
 login_email=driver.find_element(By.ID,"reg_email")
-login_email.send_keys("qwerty25@gmail.com")
+login_email.send_keys(email)
 login_passw=driver.find_element(By.ID,"reg_password")
 login_passw.send_keys("pass1234567!@")
 submit_btn=driver.find_element(By.NAME,"register")
 submit_btn.click()
 time.sleep(1)
 
-dashboard=driver.find_element(By.CSS_SELECTOR,"#post-9 > div > div > nav > ul > li.woocommerce-MyAccount-navigation-link.woocommerce-MyAccount-navigation-link--dashboard.is-active > a")
 # If the registration was successful, then print "Success" otherwise print "Fail!"
-if dashboard.is_displayed:
+if driver.find_element(By.CSS_SELECTOR,"#post-9 > div > div > nav > ul > li.woocommerce-MyAccount-navigation-link.woocommerce-MyAccount-navigation-link--dashboard.is-active > a").is_displayed:
```

Comment:
* To make it more readable you should create a variable to find element.
```
nav_link = driver.find_element(.....)
if nav_link.is_displayed():
    <code>
```
* I think you have to use `.is_displayed()` not `.is_displayed`